### DISCUSSION TITLE
pool change

### DISCUSSION THREAD

**Discussion by Trivisuko** [Chia-Network/chia-blockchain#8269] [Support]
hello, I have NFT plots created with smart contract for the ecochia pool and I want to change the pool, but when changing it tells me that I have 0 plots farming,
can you help me?
Thanks.
![pool](https://user-images.githubusercontent.com/86140843/131258990-233c1169-cce3-4a57-a68d-897c08c1afac.png)

**Comment by loppefaaret** [comment]
im assuming youre are referring to the secondaru plotnft in the picture, the Yellow Woodpecker ? how many plots does it normally have ? and there can in some circumstances take up to 10 minutes for it to update the number in the GUI

**Reply by Trivisuko** [reply]
I understand that when the pool is changed it is not immediate, it takes a while until the plots are recognized by the pool?

**Reply by loppefaaret** [reply]
when you leave a pool, it can take up to 100 blocks to take effect

**Comment by BrandtH22** [comment]
@Trivisuko , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
